articles/tutorials/web-apps-vs-web-apis-cookies-vs-tokens.md

```diff
@@ -16,6 +16,6 @@ description: This page compares web applications to web APIs and cookies vs. Tok
 
 * For both approaches you can get the **same amount of information from the user**. That's controlled by the `scope` parameter sent in the login request (either using the [Auth0Lock](/lock), our [JavaScript library](https://github.com/auth0/auth0.js) or a plain link). The `scope` is a parameter of the `.signin({scope: 'openid name email'})` method which ends up being part of the querystring in the login request. You can get more details about this in the [Scopes Documentation](/scopes).
 
-* By default we use `scope=openid` in **token-based authentication** to avoid having a huge token (since the token will be traveling in the `Authorization` header for each request as mentioned above). You can control the attributes that you want to get in the token by doing `scope=openid name email another_attribute`.
+* By default we use `scope=openid` in **token-based authentication** to avoid having a huge token. You can control the attributes that you want to get in the token by doing `scope=openid name email another_attribute`.
```

Comment:
clients can only request [standard oidc claims](http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) as scope values. from the total set, we need to double check those that auth0 supports (i think all but i need to double check)